extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -126,6 +126,10 @@ This attribute is present when the sidebar is open.
 
 This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
 
+##### data-close-button-aria-label attribute
```

Comment:
```
##### data-close-button-aria-label**

Optional string used as ARIA label for close button.
```

for consistency with other places that have this attribute (e.g. amp-image-lightbox)

also please move this above the ` ##### common attributes`